src/selectors/candidate_filters.js

```diff
@@ -0,0 +1,13 @@
+const { createSelector } = require('reselect')
+
+const getFieldAnalysis = require('./field_analysis')
+
+const getCandidateFilters = createSelector(
```

Comment:
This intentionally bypasses the synthetic “best fields” because if it includes them, it will allow them to be “removed” from the list of filters but won’t actually do anything since they’re not actually marked.

The UX would be to show 2 checked filters. You could check an additional one, but then the original 2 would disappear, showing only the new filter. If you tried to uncheck either of the 2 checked ones (if they’re the only ones available), nothing would happen.